Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1361,6 +1364,50 @@ extension Tensor {
         "Axes must have rank 0 or rank 1; axes has rank \(axes.rank) with values \(axes.scalars).",
         file: file,
         line: line)
-    return areAxesInRange(axes.scalars)
+    return isInRange(axes: axes.scalars)
```

Comment:
You didn't address the clash between the precondition check and the comment that I noted in https://github.com/tensorflow/swift-apis/pull/673#discussion_r394651649

Reply:
Sorry! I appear to have missed that note. I believe I've fixed this by removing the extra "is..." overload. This should take either a rank 0 (scalar) or rank 1 (vector) tensor, which was the cause for the change in behavior from before (as it was ambiguous).